app/models/order_item.rb

```diff
@@ -26,17 +26,6 @@ class OrderItem < ApplicationRecord
   has_many :progress_messages
   after_initialize :set_defaults, unless: :persisted?
 
-  NON_DATE_ATTRIBUTES = %w(order_id service_plan_ref portfolio_item_id state service_parameters provider_control_parameters external_ref)
-  DATE_ATTRIBUTES     = %w(created_at ordered_at completed_at updated_at)
```

Comment:
I wasn't aware that we were filtering specific columns before, not sure if sending them all is what we want either. @mkanoor should a mechanism be added to check for the above constants on a class and use them if they're declared?

Reply:
The ideal solution is to look at the openapi.json for the object and only send out the attributes that are needed. I think that's what @bdunne is doing in topology